corehq/apps/export/static/export/js/customize_export.js

```diff
@@ -1,4 +1,4 @@
-hqDefine('export/js/customize_export_new', function() {
```

Comment:
I'm a little hesitant about this change - there's still old export stuff and new export stuff, and I think it's useful to keep it obvious that these files belong to the latter.

Reply:
Okay, I'm happy to revert this commit.